packages/create-codes/src/helpers/prompt.ts

```diff
@@ -11,7 +11,7 @@ export type UserInputTests = {
 
 export type UserInput = {
   jsLibrary: JSLibrary;
-  apiSolution: string;
+  apiSolution: "restful" | "graphql";
```

Comment:
Would it be better to create a `type` for `"restful" | "graphql"`?
I think that would make it cleaner.

Reply:
Sure, no problem!